petab/v1/sampling.py

```diff
@@ -26,84 +26,10 @@ def sample_from_prior(
     """
     # unpack info
     p_type, p_params, scaling, bounds = prior
+    prior_cls = Distribution._type_to_cls[p_type]
```

Comment:
Yes. I'd like to decrease the API surface. Unless there is a clear use case, I'd consider things private to simplify refactoring. Here, I don't see a clear use case yet.

Reply:
Makes sense, my question was rather because `_type_to_cls` is used outside of `Distribution` here, but fine to use `_*` for things that can be used anywhere in the library but not "outside" the library, e.g. in user code.

Reply:
It's completely gone by now anyways, will update in a sec.